ga4gh/backend.py

```diff
@@ -413,6 +429,18 @@ def getVariantSet(self, id_):
         dataset = self._getDatasetFromCompoundId(id_)
         return self.runGetRequest(dataset.getVariantSetIdMap(), id_)
 
+    def getVariant(self, id_):
+        """
+        Returns a variant with the given id
+        """
+        compoundId = variants.CompoundVariantId(id_)
+        dataset = self._getDatasetFromCompoundId(compoundId.datasetId)
+        variantSet = _safeMapQuery(
```

Comment:
We could, but then we'd have to either have something like `_safeMapQuery` accessible in `datasets.py` as well, or have a try/catch block here, the later of which I don't like much.  Where should `_safeMapQuery` live, do you think?  It's a very general method...